### PR TITLE
Updates to address changes in Patient models

### PR DESCRIPTION
**Pull request by hackrm**
The CQM-Models repository is being updated to store patient cqm and qdm data in separate models (https://github.com/projecttacoma/cqm-models/pull/69). This PR updates CQM-Converter to coincide with the restructuring of CQM-Models. 

**Before merging:
- [x] The 'bonnie_bundle to CQM Measure model conversion' PR must be merged: #21 
- [x] Update the Gemfile to reference correct branches & remove debugging statements

Pull requests into cqm-converter require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1872
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass
- [x] If applicable, the library version number in `cqm-converter.gemspec` has been updated
- [x] Update cqm-execution test fixtures using update_cqm_execution_fixtures.sh script from server-scripts repo

**Bonnie Reviewer:**

Name: Zach Lister
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**~Cypress~Other Bonnie Reviewer:**

Name: @losborne
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ __Note__: This rake task is not intended for anything other than experimentation
 
 Example:
 ```
-bundle exec rake cqm:hds:to_qdm RECORD=spec/fixtures/hds/records/eh/1.json
+bundle exec rake cqm:hds:to_cqm RECORD=spec/fixtures/hds/records/eh/1.json
```
Comment (losborne) [inline README.md]: Line 41 should also be updated to CQM from QDM in this file.

File: spec/cqm/converter/hds_record_spec.rb [hunk 2]
```diff
@@ -8,14 +8,13 @@
 
   it 'converts HDS to QDM and maintains id' do
```
Comment (zlister) [inline spec/cqm/converter/hds_record_spec.rb]: converts HDS to CQM and maintains id* is probably more accurate

Comment (zlister) [inline spec/cqm/converter/hds_record_spec.rb]: 'converts HDS to CQM and maintains QDM.id' may be slightly more accurate? I'm not totally sure, but that is how it looks to me

File: spec/cqm/converter/hds_record_spec.rb [hunk 3]
```diff
@@ -39,49 +38,49 @@
 
   it 'converts HDS EH 1 to QDM EH 1 properly' do
```
Comment (zlister) [inline spec/cqm/converter/hds_record_spec.rb]: Looks like several of these test names may need to be adjusted depending on what is being tested.

File: spec/cqm/converter_spec.rb [hunk 4]
```diff
@@ -24,9 +24,9 @@
       # Read in fixture as an HDS Record.
       hds_record1 = Record.new.from_json(File.read(record_path))
       # Convert the HDS Record to a QDM Patient.
```
Comment (zlister) [inline spec/cqm/converter_spec.rb]: Comment change from QDM to CQM

File: spec/cqm/converter_spec.rb [hunk 5]
```diff
@@ -24,9 +24,9 @@
...
-      qdm_record = @hds_record_converter.to_qdm(hds_record1)
+      cqm_record = @hds_record_converter.to_cqm(hds_record1)
       # Convert the QDM Patient back to an HDS Record.
```
Comment (zlister) [inline spec/cqm/converter_spec.rb]: QDM to CQM change here too
